src/Modules/OrchardCore.Commerce/Models/StripePaymentPart.cs

```diff
@@ -0,0 +1,10 @@
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentManagement;
+
+namespace OrchardCore.Commerce.Models;
+
+public class StripePaymentPart : ContentPart
+{
+    public TextField PaymentMethodId { get; set; } = new();
+    public TextField PaymentIntentId { get; set; } = new();
```

Comment:
> Do we really want these to be editable from the admin UI? I see no reason for that.

It's not that I want to make it editable, but making it read-only seems like an unnecessary effort for no gain.

> Also, PaymentIntentId is null when there is no 3DS authentication. This is intended, because there is no "intent" without 3DS, but makes the field a bit more unnecessary on admin.

Isn't that a loss of information we will regret later?